spec/lib/schedule_spec.rb

```diff
@@ -187,16 +187,6 @@
     end
   end
 
-  context "prison without unbookable dates" do
-    let :prison do
-      Rails.configuration.prison_data['Oakwood'].dup
-    end
-
-    it "should return a list of dates" do
-      subject.dates(start_date = Date.new(2014, 12, 4)).first.should be_nil
-    end
-  end
-
```

Comment:
Given that we're not removing unbookable prisons after all, can we add this back?